README.md

```diff
@@ -18,7 +18,7 @@ Additional description about the project and its features.
 
 [Live Demo Link](https://zairdon20.github.io/JOSPHATS-FIRST/)
 
-
+[Online view link ](https://zairdon20.github.io/JOSPHATS-FIRST/)
```

Comment:
- [ ] [OPTIONAL] You need only one link! I think `Live Demo Link` is enough for it! 😊
```suggestion
```

Reply:
Thanks for the advice